src/System.Security.Permissions/src/System/Security/SecurityElement.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.s
+
+namespace System.Security
+{
+    public sealed partial class SecurityElement
+    {
+        public SecurityElement(string tag) { }
+        public SecurityElement(string tag, string text) { }
+        public System.Collections.Hashtable Attributes { get; set; }
+        public System.Collections.ArrayList Children { get; set; }
+        public string Tag { get; set; }
+        public string Text { get; set; }
+        public void AddAttribute(string name, string value) { }
+        public void AddChild(System.Security.SecurityElement child) { }
+        public string Attribute(string name) { return default(string); }
+        public System.Security.SecurityElement Copy() { return default(System.Security.SecurityElement); }
```

Comment:
Maybe return `this` instead?  I'm not sure how we decide on if we should return null or an actual object.  We seem to do different things in different places.  /cc @stephentoub in case he has any thoughts.